src/coreclr/tools/aot/ILCompiler.Diagnostics/PerfMapWriter.cs

```diff
@@ -16,13 +16,15 @@ public class PerfMapWriter
         public const int LegacyCrossgen1FormatVersion = 0;
 
         public const int CurrentFormatVersion = 1;
-        
+
+        // These pseudo RVAs are used by other tools, please only append to this.
         public enum PseudoRVA : uint
         {
-            OutputGuid         = 0xFFFFFFFF,
```

Comment:
I suggest moving this to a file which *only* contains constants that are expected to be preserved for file format purposes. However, as these constants are tied to the PerfMap format, if you do not wish to do so, that's alright too.